src/utils/HubDevice.hxx

```diff
@@ -47,10 +48,18 @@ template <class Data> class FdHubWriteFlow;
 class FdHubPortBase : public FdHubPortInterface, private Atomic
 {
 public:
+#ifdef ESP32 // TODO: shrink these if possible
```

Comment:
If we need these ifdefs then maybe we should make these from constants into DECLARE_CONST type parameters. You can file an issue against me and I can migrate them.

We already have a parameter called main_thread_stack_size or similar.

Reply:
https://github.com/bakerstu/openmrn/issues/255